File: FuncamentosProgramacao/CapituloTres/AtividadeQuatro.cs
namespace FuncamentosProgramacao.CapituloTres;

public class AtividadeQuatro
{
    public static void CalculaMedia()
    {
        double notaUm;
        double notaDois;
        double mediaPonderada;

        Console.Write("Digite o valor da primeira nota: ");
        notaUm = double.Parse(Console.ReadLine()!);
        Console.Write("Digite o valor da segunda nota: ");
        notaDois = double.Parse(Console.ReadLine()!);

        mediaPonderada = ((notaUm * 2) + (notaDois * 3)) / 5;

        Console.WriteLine($"A média ponderada das notas {notaUm} e {notaDois} = {mediaPonderada:F}");
    }
}

File: fundamentosProgramacaoJava/src/capitulo/tres/AtividadeQuatro.java
package capitulo.tres;

import java.util.Scanner;

public class AtividadeQuatro {

    public static void main(String[] args) {

        float notaUm;
        float notaDois;
        float mediaPonderada;
        Scanner leia = new Scanner(System.in);

        System.out.print("Digite o valor da primeira nota: ");
        notaUm = leia.nextFloat();
        System.out.print("Digite o valor da segunda nota: ");
        notaDois = leia.nextFloat();

        mediaPonderada = ((notaUm * 2) + (notaDois * 3)) / 5;

        System.out.println("A media ponderada das notas e: " + mediaPonderada);

        leia.close();

    }

}
